File: tests/fixtures/corpus/delphi/playground.pas
program Demo;

procedure Greet(const Name: string);
begin
  WriteLn(Format('Hello, %s!', [Name]));
end;

function Add(const A, B: Integer): Integer;
begin
  Result := A + B;
end;

function Multiply(const A, B: Integer): Integer;
begin
  Result := A * B;
end;

begin
  Greet('World');
  WriteLn(Add(2, 3));
  WriteLn(Multiply(2, 3));
end.

File: tests/fixtures/corpus/vbnet/playground.vb
Module HelloWorld
    Sub Greet(name As String)
        Console.WriteLine($"Hello, {name}!")
    End Sub

    Function Add(a As Integer, b As Integer) As Integer
        Return a + b
    End Function

    Function Multiply(a As Integer, b As Integer) As Integer
        Return a * b
    End Function
End Module
